### ISSUE TITLE
Book 05: Sentence 009

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#47]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/9f5ef51b-2642-4a9a-98a2-103747ed4b31)

```
# sent_id = Book_05_-_The_Omens.txt:9
# text = Auh injn, ca oneltico: aco ie tocommaujcoz, in quenamj momaceval: ma ximellaquaoa, ma titlaocux, ixqujch motlapal: oc motepitziollo, oc mochoqujz, motlaocul ipan xonie.
# text[norm] = Auh inin, ca oneltico: azo ye tocommahuizoz, in quenami momacehual: ma ximellacuahua, ma titlaocox, ixquich motlapal: oc motepitzyollo, oc mochoquiz, motlaocol ipan xonye.
# text[orig] = A·uh¶injn·,·caoneltico·:·aco·ie·tocommaujcoz·,·inquenamj·momaceval·:·maximella¶quaoa·,·matitlaocux·,·ixqujch·motla¶pal·:·oc·motepitziollo·,·oc·mochoqujz·,·motlaocul·ipan·xonie·.
# text[osp] = Y esto, no soy yo el que te pongo espanto, o miedo: que el mismo señor dios quiso, que esto te acontesciesse, y vieniesse sobre ti: y no ay que culpar al animal, por que el, no sabe lo que se haze, porque carece de entendimiento de rrazon: y tu pobrezito, no deues deculpar a nadie: porque el signo en que naciste, tiene consigo estos azares, ya venido: agora auerificar se enti, la maldad del signo en que naciste: esfuerçate porque por esperiencia lo sentiras: mira que tengas buen animo para sufrirlo: y entre tanto llora, y haz penitencia.
# text[osp_norm] = Y esto, no soy yo el que te pongo espanto, o miedo: que el mismo señor dios quiso, que esto te aconteciese, y viniese sobre ti: y no hay que culpar al animal, por que él, no sabe lo que se hace, porque carece de entendimiento de razón: y tú pobrecito, no debes de culpar a nadie: porque el signo en que naciste, tiene consigo estos azares, ya venido: ahora a verificarse en ti, la maldad del signo en que naciste: esfuérzate porque por experiencia lo sentirás: mira que tengas buen ánimo para sufrirlo: y entre tanto llora, y haz penitencia.
# references = A&D/161:2
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 36.36%
1	Auh	auh	CCONJ	_	_	5	_	_	_
2	injn	inin	DET|PRON	_	_	5	_	_	_
3	,	,	PUNCT	_	_	5	punct	_	_
4	ca	cah|ca	VERB	_	_	5	_	_	_
5	oneltico	eltia	VERB	_	Aspect=Prior|Directional=Ven|Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	_
6	:	:	PUNCT	_	_	9	punct	_	_
7	aco	azo	ADV	_	_	9	_	_	_
8	ie	ye	ADV	_	_	9	_	_	_
9	tocommaujcoz	mahuizoa	VERB	_	Movement=And|Number[obj]=Sing|Number[subj]=Sing|Person[obj]=3|Person[subj]=2|Subcat=Tran|Tense=Fut|VerbForm=Fin	5	_	_	_
10	,	,	PUNCT	_	_	13	punct	_	_
11	in	in	DET|SCONJ	_	_	13	_	_	_
12	quenamj	quenami	ADV	_	_	13	_	_	_
13	momaceval	macehualli	NOUN	_	Number[psor]=Sing|Person[psor]=2|State=Pos	9	_	_	_
14	:	:	PUNCT	_	_	16	punct	_	_
15	ma	ma	ADV	_	_	16	_	_	_
16	ximellaquaoa	ellacuahua	VERB	_	Mood=Imp|Number[subj]=Sing|Person[subj]=2|Reflex[obj]=Yes|Subcat=Tran|VerbForm=Fin	5	_	_	_
17	,	,	PUNCT	_	_	19	punct	_	_
18	ma	ma	ADV	_	_	19	_	_	_
19	titlaocux	tlaocoya	VERB	_	Number[subj]=Sing|Person[subj]=2|Subcat=Intr|Tense=Past|VerbForm=Fin	16	_	_	_
20	,	,	PUNCT	_	_	22	punct	_	_
21	ixqujch	ixquich	NOUN	_	NounType=Relat	22	_	_	_
22	motlapal	tlapalli	NOUN	_	Number[psor]=Sing|Person[psor]=2|State=Pos	19	_	_	_
23	:	:	PUNCT	_	_	25	punct	_	_
24	oc	oc	ADV	_	_	25	_	_	_
25	motepitziollo	tepitzyollotl	NOUN	_	Number[psor]=Sing|Person[psor]=2|State=Pos	31	_	_	_
26	,	,	PUNCT	_	_	28	punct	_	_
27	oc	oc	ADV	_	_	28	_	_	_
28	mochoqujz	choquiztli	NOUN	_	Number[psor]=Sing|Person[psor]=2|State=Pos	25	_	_	_
29	,	,	PUNCT	_	_	30	punct	_	_
30	motlaocul	tlaocolli	NOUN	_	Number[psor]=Sing|Person[psor]=2|State=Pos	25	_	_	_
31	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	32	obl	_	_
32	xonie	cah	VERB	_	Mood=Imp|Movement=And|Number[subj]=Sing|Person[subj]=2|Subcat=Intr|VerbForm=Fin	16	_	_	_
33	.	.	PUNCT	_	_	5	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> Y esto es lo que ha sido verifi­cado. Pudiera ser que te maravilles por lo que mereces. Esfuérzate, no estés triste; ¡toda tu fuerza! Esfuerza tu corazón; pasa por en­cima de tu lloro y de tu tristeza.

**Comment by ftyers** [comment]
Questions:
- What's up with *ma*? Do we want `AUX`/`aux`,  `ADV`/`advmod` or `PART`/`discourse`  ? If it is `AUX`/`aux` then what would be the [auxiliary function](https://quest.ms.mff.cuni.cz/udvalidator/cgi-bin/unidep/langspec/specify_auxiliary.pl?lcode=nci)? 
- What is *xonye* is it a form of *yohui* or of *cah* ?
- What is the structure of *ixquich* + `NOUN`? 
-

**Comment by Lguyogiro** [comment]
> What's up with ma? Do we want AUX/aux, ADV/advmod or PART/discourse ? If it is AUX/aux then what would be the [auxiliary function](https://quest.ms.mff.cuni.cz/udvalidator/cgi-bin/unidep/langspec/specify_auxiliary.pl?lcode=nci)?

Yeah I think `AUX` makes sense, as it marks the optative mood. iirc that is not one of the existing UD aux functions.

> What is xonye is it a form of yohui or of cah ?

I'm pretty sure _cah_. It patterns with the future tenses of `cah`, e.g. _tonyez_ "you will be." (also interesting note:  a current remnant of this form in nhi: _i_ as in _amo ma i ohcon_ "may it not be so")

> What is the structure of ixquich + NOUN?

I kind of like treating it as a quantifier: DET NOUN and `det(NOUN, ixquich)`

**Comment by ftyers** [comment]
Re *ixquich* (Andrews, p.132): 
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/a5980a64-fb0b-4eab-b17b-98c9035d3fe1)

**Comment by Lguyogiro** [comment]
here's an updated attempt...I'm not super happy with all of this, but I tried to add possible rels in most places.

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/ff7c98a0-5afb-44ea-a7c4-e4cd0ae944d3)

**Comment by ftyers** [comment]
Here is a suggestion:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/07250c6c-6a69-4ec5-95c3-07663765d8a0)

* For *quenami*, Andrews (p.93) has it as a contraction of *quen* + a verb *ami* (meaning "to exist"), I'm not sure if this works, if *momacehual* should be `nsubj` or `xcomp`...
* For the complement of *tocommahuizoz* (this is transitive?) [ti-c-on-mahuizo-z] then I've got `ccomp`
* The part from *ximellacuahua* to *xonye* is a bit confused, which parts are conjoined, and which parts go with which verbs...

**Comment by Lguyogiro** [comment]
> For quenami, Andrews (p.93) has it as a contraction of quen + a verb ami (meaning "to exist"), I'm not sure if this works, if momacehual should be nsubj or xcomp...

hmm..Molina, Carochi, etc. all consider _quenami_ an `ADV`. I don't doubt that it has its etymology in _quen_ + _ami_. it is never written separately as far as I can tell in the codex.  Also, in Launey there is an example of the plural (in a present tense clause), _quenamihqueh_ which I think is not what we would expect if _ami_ were a verb.

**Comment by Lguyogiro** [comment]
I think ipan xonye is copular, so the head should be _ipan_. e.g. if this were not optative, something like _ocatca ipan_ _it was above (it)_ we would also say _catqui_ is `AUX` with `cop` right?

**Comment by Lguyogiro** [comment]
I'm not clear why we do `ccomp` to _quenami_ still.

**Comment by ftyers** [comment]
> > For quenami, Andrews (p.93) has it as a contraction of quen + a verb ami (meaning "to exist"), I'm not sure if this works, if momacehual should be nsubj or xcomp...
> 
> hmm..Molina, Carochi, etc. all consider _quenami_ an `ADV`. I don't doubt that it has its etymology in _quen_ + _ami_. it is never written separately as far as I can tell in the codex. Also, in Launey there is an example of the plural (in a present tense clause), _quenamihqueh_ which I think is not what we would expect if _ami_ were a verb.

I think I mostly agree. On the other hand, we wouldn't expect `ADV` to have a plural... If it is `ADV`, then what is the structure of *in quenami momacehual* ? If it is an adverbial clause then one of those bits has to be the predicate...

> I'm not clear why we do ccomp to quenami still.

It's `ccomp` if the *in quenami momacehual* is the object complement of *tocommahuizoz* (if that is coindexed with the -c- object).

If it isn't, perhaps *in quenami momacehual* is `advcl`, then it would not be coindexed, but what would the structure of the adverbial clause be?

> I think ipan xonye is copular, so the head should be ipan. e.g. if this were not optative, something like ocatca ipan it was above (it) we would also say catqui is AUX with cop right?

Probably yes, but it's just icky XD

**Comment by Lguyogiro** [comment]
- coming back to _quenami_, and having reread launey's treatment of it (26.11), I'm on board with the (_quen ami_, `ADV VERB`) analysis, since, in this construction _ami_ can also take subject prefixes e.g. _quen tami_ "what are you"

- re ickiness of _ipan xonye_, I think we should make _ipan_ the head....if later we want to slightly modify the guidelines around copulae and locatives I'm ok with that, but `ickiness > inconsistency` imho.

**Comment by ftyers** [comment]
How about this:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8f95f45f-3295-4953-9589-4f00a6772787)

Note the coordination structure (going with López Austin and not A&D), in many cases there are kind of sequences that invert, viz. *A oc B, oc C, D*.

**Comment by ftyers** [comment]
Yeah, that's a good point. But there is some weirdness in the translation because there are two verbs (in different places) that don't appear in the original and a negation which is not there:

| `nci`                          | `spa`         | `eng`                  |
|---------------------------------|---------------|-------------------------|
|  *ma ximellacuahua,* | Esfuérzate| Take courage |
|  *ma titlaocox,* |  **no** estés triste|   be **not** sad|  
|  *ixquich motlapal:*  |  toda tu fuerza|  **use** all thy force|  
|  *oc motepitzyollo,* |  **esfuerza** tu corazón|   hardness of thy heart|  
|  *oc mochoquiz,* |  tu lloro|   thy tears|  
|   *motlaocol* |  tu tristeza|   thy grief|  
|  *ipan xonye.* |  pasa por encima de|   be above|

**Comment by ftyers** [comment]
> hmm so are you suggesting that motlapal should get promoted to object of a different verb since presumably its verb is missing? In that case wouldn't something like "dislocated" be better?

Hmm no, I just thought that maybe there is something missing in the translation(s), or there is some other interpretation... 

> I actually prefer the lopez austin translation in this part (¡Toda tu fuerza!), which would mean `parataxis` I think

Yep, but then what do we do with *oc motepitzyollo* ?

**Comment by Lguyogiro** [comment]
Maybe we group _ixquich motlapal_ and _oc motipetzyollo_ with `conj`, like `parataxis(..., motlapal)` and `conj(motlapal, motipetzyollo)`, and then the other two in the sequence of nouns are conjoined dependents of `ipan`? That would at least track semantically I think (e.g. "use all your force (and) the hardness of your heart, be above sadness/tears/etc" kind of makes sense I guess??).

**Comment by ftyers** [comment]
Like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/eb6873b4-f6d6-4873-9440-df7b3a4b38e9)
